tests/networkflows_test.go

```diff
@@ -23,27 +32,24 @@ func hasEdges(graph *v1.NetworkGraph) bool {
 	return false
 }
 
-func TestStackroxNetworkFlows(t *testing.T) {
-	t.Parallel()
-
-	conn := centralgrpc.GRPCConnectionToCentral(t)
+func (s *networkFlowTestSuite) TestStackroxNetworkFlows() {
+	conn := centralgrpc.GRPCConnectionToCentral(s.T())
 
 	clustersService := v1.NewClustersServiceClient(conn)
 	ctx, cancel := context.WithTimeout(context.Background(), time.Minute)
 
 	var clusters *v1.ClustersList
 	var err error
-	for retries := 20; retries > 0; retries-- {
+
+	s.Eventually(func() bool {
 		clusters, err = clustersService.GetClusters(ctx, &v1.GetClustersRequest{})
-		if err == nil && len(clusters.GetClusters()) > 0 {
-			break
-		}
-		time.Sleep(5 * time.Second)
-	}
+		return err == nil && len(clusters.GetClusters()) > 0
+	}, 2*time.Minute, 5*time.Second)
+
 	cancel()
 
-	require.NoError(t, err)
-	require.NotEqual(t, 0, len(clusters.GetClusters()))
+	require.NoError(s.T(), err)
+	require.NotEqual(s.T(), 0, len(clusters.GetClusters()))
```

Comment:
[nit] If you're using the `suite`, you could just simply call:
```suggestion
	s.Require.NoError(err)
	s.Require.NotEqual(0, len(clusters.GetClusters()))
```